includes/class-wc-session-handler.php

```diff
@@ -63,6 +66,7 @@ public function __construct() {
 		}
 
 		$this->_data = $this->get_session_data();
+		$this->_data_org = serialize($this->_data);
```

Comment:
We can, but I wouldn't advise it.

serialize/unserialize is built for serializing all PHP variable values (except resources AFAIK).
json_encode/json_decode should not be aware of PHP specifics, maybe this results in different strings which actually represent the same value.
I'm especially worried about objects.

As the serialized string does not hit the database or travel outside of this class, the data format of the string should not be a concern. But we can try it and see if it works.

I will make the change and test that implementation (hopefully tomorrow).
And report my findings here.